test/ci/create_aws_vm.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Create a VM
+  ec2:
+    region: "{{ aws_region }}"
+    key_name: "{{ aws_key }}"
+    instance_type: "{{ outer_item.aws_flavor }}"
+    image: "{{ outer_item.aws_image | default(aws_image) }}"
+    wait: yes
+    group: "{{ outer_item.aws_security_group }}"
+    count: 1
+    vpc_subnet_id: "{{ aws_subnet }}"
+    assign_public_ip: yes
+    instance_tags:
+      Name: "{{ outer_item.name_template }}-{{ item }}"
+      "kubernetes.io/cluster/{{ aws_cluster_id }}": "true"
+  register: ec2
+  with_sequence: start=1 end={{ outer_item.amount }}
+
+- name: Attach the volume for gluster
+  ec2_vol:
+    instance: "{{ item.instances.0.id }}"
+    name: "{{ item.instances.0.tags.Name }}-volume"
+    device_name: /dev/sdc
+    volume_size: 50
+    delete_on_termination: yes
+    region: "{{ aws_region }}"
+  when: "'glusterfs' in outer_item.groups"
+  with_items: "{{ ec2.results }}"
+
+- name: Attach the volume for Atomic
+  ec2_vol:
+    instance: "{{ item.instances.0.id }}"
+    name: "{{ item.instances.0.tags.Name }}-docker-volume"
+    device_name: /dev/sdb
+    volume_size: 50
+    delete_on_termination: yes
+    region: "{{ aws_region }}"
+  when: is_atomic
+  with_items: "{{ ec2.results }}"
+
```

Comment:
i suspect you do need to tag your volumes with same `kubernetes.io/cluster/` key and value above

Reply:
you want this especially if you planning to use PVs ... just saying

Reply:
This is a volume for Atomic's docker storage, it doesn't know about clusters and such. In any case it seems the bot on our instance is deleting those correctly without any tags